### ISSUE TITLE
liste tournois dans schedule

### ISSUE THREAD

**Issue by mupfpilou** [Futurolan/ga-front#39]
tous les tournois apparaissent, sans distinction edition, compliqué du coup
faudrait soit préselect l'édition, soit afficher edition+tournoi

**Comment by bmagic** [comment]
Yep j'avais déjà tenté de faire cela, mais bon il y avait un bug dans drupal qui devrait être fixe. Bref faut que je teste à nouveau.

**Comment by mupfpilou** [comment]
on laisse ouvert du coup pour pas oublier ?
